src/digitalmarketplace/components/alert/template.njk

```diff
@@ -0,0 +1,10 @@
+<div data-module="dm-alert" class="dm-alert dm-alert--{{ params.type }}{%- if params.classes %} {{ params.classes }}{% endif %}"{%- for key, value in params.attributes %} {{key}}="{{value}}"{% endfor %} tabindex="0">
```

Comment:
This could probably do with being split over one or more lines to improve readability

Reply:
What are the `data-module` and `tab-index` attributes for?

Reply:
> What are the `data-module` and `tab-index` attributes for?

`tab-index` is for focus on tab. `data-module` is for tracking I think.

Reply:
> This could probably do with being split over one or more lines to improve readability

With the next commit.

Reply:
`data-module` is a convention that govuk-frontend uses to namespace components for js/css hooks. This means that if govuk-frontend were to create their own alert component we could use them together on the same page without there being issues with components affecting other parts of the page.

Reply:
Do we need to have the `data-module` attribute if we don't have any CSS/JS hooks?

Reply:
Yes, I think it's a good pattern to follow, we might not need it here but we will for other components and it keeps components consistent and in line with govuk-frontend standards

Reply:
Definitely keep `data-module` unless it's causing backwards compatibility problems. It will help us when we revamp our JS...